unskript-client.py

```diff
@@ -0,0 +1,889 @@
+#!/usr/bin/env python
+#
```

Comment:
why is this duplicated, i see one in .github/script as well

Reply:
There are three files in the .github/scripts. 
1. unskript-ctl.sh (The shell script that is the wrapper around .py file)
2. unskript-client.py (This is the main python file that implements the business logic)
3. db_utils.py (This is a utility python file that is used by unskript-client.py)